src/Libraries/Microsoft.Extensions.AI/ChatCompletion/ChatClientStructuredOutputExtensions.cs

```diff
@@ -19,6 +16,13 @@ namespace Microsoft.Extensions.AI;
 /// </summary>
 public static class ChatClientStructuredOutputExtensions
 {
+    private static readonly JsonSchemaInferenceOptions _inferenceOptions = new()
+    {
+        IncludeSchemaKeyword = true,
+        DisallowAdditionalProperties = true,
+        IncludeTypeInEnumSchemas = true
```

Comment:
These values differ from the JsonSchemaInferenceOptions defaults. Do we have the right defaults? Or is this just a special case?

Reply:
Good question. It isn't clear to me if we need to adopt some of these defaults in AIFunctionFactory. The current divergences reflect the status quo.